cmake/patches/fizz.v2020.05.18.00.cmake

```diff
@@ -0,0 +1,18 @@
+36c36
+< find_package(folly CONFIG REQUIRED)
+---
+> find_package(Folly  REQUIRED)
+46a47,54
+> 
+> if(NOT APPLE)
+>   if(NOT WIN32)
```

Comment:
Curious whats the story behind this?

Reply:
all-load ( if memory serves me ) is part of the conditional to have the same functionality in OSX, and I don't believe WIN32 suffers from this issue -- the expressions that are there are linux only. I have a habit of using this on projects. I'd be okay removing it since we probably won't build on Windows . At the very least if someone builds on OSX we shouldn't fail but I'll just use if (LINUX) since I wanted to push OSX validation to a separate ticket.

Reply:
No worries, was just curious. ya, if (LINUX) would be simpler.